core/koin-test/build.gradle

```
@@ -15,18 +15,22 @@ kotlin {
         macosX64("native")
     }else{
         macosX64()
+        macosArm64()
         iosArm32()
         iosArm64()
         iosX64()
+        iosSimulatorArm64()
         linuxX64()
         linuxArm32Hfp()
         linuxMips32()
         watchosArm32()
         watchosArm64()
         watchosX86()
         watchosX64()
+        watchosSimulatorArm64()
         tvosArm64()
         tvosX64()
+        tvosSimulatorArmX64()
```

I believe this is a typo; was causing issues for me building off this branch
```suggestion
        tvosSimulatorArm64()
```

---

Oops, I really need to stop using GitHub as an IDE